src/test/java/org/apache/commons/lang3/function/ConsumersTest.java

```diff
@@ -46,4 +51,23 @@ public void testNop() {
         Consumers.nop().accept("");
     }
 
+    /**
+     * Tests {@link Consumers#accept(Object, Consumer)}.
+     */
+    @Test
+    public void testAccept() {
+        final StringBuilder builder = new StringBuilder("foo");
```

Comment:
Don't use mock testing here, it makes the test too obtuse; this is not a hard feature to test IMO.

Reply:
Thank you for your feedback.
I modified my code to not use mock testing.